jenkins.bash

```diff
@@ -14,29 +14,37 @@ cloneorpull() {
 }
 venv() {
         if ! test -d "$1" ; then
-                virtualenv "$1"
+                virtualenv -p python${PYTHON_VERSION} "$1"
         fi
 	# shellcheck source=/dev/null
         source "$1"/bin/activate
 }
-git clean --force -d -x || /bin/true
-cloneorpull common-workflow-language https://github.com/common-workflow-language/common-workflow-language.git
-venv cwltool-venv
-docker pull node:slim
-export PIP_DOWNLOAD_CACHE=/var/lib/jenkins/pypi-cache/
-pip install -U setuptools wheel pip
-python setup.py install
-pip install "cwltest>=1.0.20160825151655"
-pushd common-workflow-language
-git clean --force -d -x || /bin/true
-# shellcheck disable=SC2154
-if [[ "$version" = *dev* ]]
-then
-	EXTRA="EXTRA=--enable-dev"
-fi
-./run_test.sh --junit-xml=result.xml RUNNER=cwltool -j4 DRAFT="${version}" ${EXTRA}
-CODE=$?
-popd
+
+# Test for Python 2.7 and Python 3
+for PYTHON_VERSION in 2 3
+do
+	git clean --force -d -x || /bin/true
```

Comment:
This deletes the test results on the second go through the loop. Move lines 26-28 outside?